docs/user.md

```diff
@@ -45,22 +45,7 @@ A Red Hat OpenShift 4 cluster must be [installed](https://docs.openshift.com/con
 
 ### Standard Environment <a name="standard-env"></a>
 
-A standard machine layout can be used for certification. If you run into issues with pod disruption (eviction, OOM, frequent crashes, etc) then you may want to consider the Dedicated Test Environment configuration further below. Below is a table of the minimum resource requirements for the OpenShift cluster under test:
-
-| Machine       | Count | CPU | RAM (GB) | Storage (GB) |
-| ------------- | ----- | --- | -------- | ------------ |
-| Bootstrap     | 1     | 4   | 16       | 100          |
-| Control Plane | 3     | 4   | 16       | 100          |
-| Compute       | 3     | 4   | 16       | 100          |
-
-
-*Note: These requirements are higher than the [minimum requirements](https://docs.openshift.com/container-platform/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-minimum-resource-requirements_installing-bare-metal) in OpenShift product documentation due to the resource demand of the certification tests.*
-
-### Dedicated Node for Test Environment <a name="dedicated-env"></a>
-
-If your compute nodes are at or below minimum requirements, it is recommended to run the certification environment on one dedicated node to avoid disruption of the test scheduler. Otherwise, the concurrency between resources scheduled on the cluster, e2e-test manager (aka openshift-tests-plugin), and other stacks like monitoring can disrupt the test environment, leading to unexpected results, like the eviction of plugins or certification server (sonobuoy pod).
-
-See the troubleshooting section on ways to identify you might need to use a dedicated node test environment below.
+A dedicated compute node should be used to avoid disruption of the test scheduler. Otherwise, the concurrency between resources scheduled on the cluster, e2e-test manager (aka openshift-tests-plugin), and other stacks like monitoring can disrupt the test environment, leading to unexpected results, like the eviction of plugins or certification server (sonobuoy pod).
```

Comment:
```suggestion
A dedicated compute node should be used to avoid disruption of the test scheduler. Otherwise, the concurrency between resources scheduled on the cluster, e2e-test manager (aka openshift-tests-plugin), and other stacks like monitoring can disrupt the test environment, leading to unexpected results, like the eviction of plugins or aggregator server (`sonobuoy` pod).
```